sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -1291,6 +1312,26 @@ class ContainerAPITest extends APISpec {
             .verifyComplete()
     }
 
+    def "List blobs hier options maxResults by page"() {
```

Comment:
typo. hier

Reply:
I don' think this is a typo? This is how all the other hierarchical listing tests are prefixed.

Reply:
i thought it's "tier" , maybe expanding hier would help.

Reply:
For some of them, expanding hierarchy makes the test name too long so that the resulting container names are invalid